app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+  def Show
+
+  end
+  def new
+    @post = Post.find_by_id(params[:post_id])
+  end
+
+  def create
+
+    p "here is there"
```

Comment:
please remove print statements. Rails default logs are enough. And for debugging use `binding.pry`

Reply:
Done